src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -50,32 +50,302 @@ Notifier::~Notifier (void)
   LM_T(LmtNotImplemented, ("Notifier destructor is not implemented"));
 }
 
+
+
+/* ****************************************************************************
```

Comment:
I am always a friend to creating new files and modules.
And it is true, the substitution in itself has nothing to do with notifications.
Perhaps it should be a module in common lib ...